src/components/mainNavbar.jsx

```diff
@@ -67,14 +67,14 @@ const MainNavbar = () => {
         <ul className={classes.navContainerList}>
           {isAuth ? (
             <>
-              {navItems.map((item) => {
-                return <li className={classes.navList}>{item}</li>;
+              {navItems.map((item, index) => {
+                return <li className={classes.navList} key={index}>{item}</li>;
               })}
             </>
           ) : (
             <>
-              {navItemsLink.map((item) => {
-                return <li className={classes.navList}>{item}</li>;
+              {navItemsLink.map((item, index) => {
```

Comment:
index?

Reply:
Every child in a list should have a unique "key" prop. To ignore the warning on the map.